service/awsconfig/v10/resource/cloudformation/main_stack_test.go

```diff
@@ -123,164 +123,163 @@ func TestMainGuestTemplateExistingFields(t *testing.T) {
 	}
 	newResource, err := New(cfg)
 	if err != nil {
-		t.Errorf("unexpected error %v", err)
+		t.Fatalf("unexpected error %v", err)
```

Comment:
When something goes wrong here `Error` just smashes everything onto the terminal and you ain't see shit. Now you see one issue at a time and have space left to breath.

Reply:
<3 I like the change to fatal and matches the conventions in `fmt`.